make/modules/java.base/gensrc/GensrcScopedMemoryAccess.gmk

```diff
@@ -42,6 +42,17 @@ define GenerateScopedOp
 
   $1_Type := $2
 
+  ifeq ($$($1_Type), Boolean)
```

Comment:
These changes in ScopedMemoryAccess are not strictly required -- but I imagine they simplify your implementation of the var handle impl methods. So, it's ok to leave them in.